src/flb_input_chunk.c

```diff
@@ -1499,31 +1499,6 @@ static int input_chunk_append_raw(struct flb_input_instance *in,
     flb_chunk_trace_do_input(ic);
 #endif /* FLB_HAVE_CHUNK_TRACE */
 
-    /* Update 'input' metrics */
-#ifdef FLB_HAVE_METRICS
-    if (ret == CIO_OK) {
```

Comment:
When in this spot, this check would fail because the last time `ret` is set is by a function that returns `CIO_TRUE`. `CIO_TRUE` is a `!0` macro. `CIO_OK` is 0, meaning this check would always fail when `flb_chunk_is_up` or `cio_chunk_up_force` succeeded.